modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -1225,6 +1225,22 @@ TEST_P(Test_ONNX_layers, Softmax)
     testONNXModels("softmax_unfused");
 }
 
+TEST_P(Test_ONNX_layers, Softmax_axis_0){
+    testONNXModels("softmax_axis_0");
+}
+
+TEST_P(Test_ONNX_layers, Softmax_axis_1){
+    testONNXModels("softmax_axis_1");
+}
+
+TEST_P(Test_ONNX_layers, Softmax_axis_2){
+    testONNXModels("softmax_axis_2");
+}
+
+TEST_P(Test_ONNX_layers, softmax_axis_negative){
+    testONNXModels("softmax_axis_negative");
+}
+
```

Comment:
Are these tests from conformance? If so, turn them on if they are off before. If the are missed, then add them in opencv_extra.